packages/vsce/__tests__/__e2e__/setup.sh

```diff
@@ -0,0 +1,8 @@
+
+install-extension /config/workspace/dist/cics-extension-for-zowe-3.*.vsix --extensions-dir /config/extensions
+
+mkdir -p /config/data/User
+rm /config/data/User/settings.json || true
```

Comment:
Could do a check if this file exists and run the `rm` if not, rather than having an error in the CI log